nodes/viz/viewer_draw_mk4.py

```diff
@@ -757,7 +754,7 @@ def migrate_from(self, old_node):
     def draw_property_socket(self, socket, context, layout):
         drawing_verts = socket.name == "Vertices"
         prop_to_show = "point_size" if drawing_verts else "line_width"
-        text = f"{socket.name}. {SvGetSocketInfo(socket)}"
+        text = f"{socket.name}. {str(socket.objects_number)}"
         layout.label(text=text)
```

Comment:
maybe this can be simplified?
```python
        text = f"{socket.name}. {str(socket.objects_number)}"
        layout.label(text=text)
```
to
```python

    <socketclass>
    def label_and_count(self, name=None):
        return f"{name or socket.name}. {self.objects_number}"        


    <some layout>
        layout.label(text=socket.label_and_count())
```

Reply:
Yes. But I'd rather add such changes in separate PR because this one does not make any conceptual changes nor suggest any improvements in this area.

Probably nicer way would be to add number of objects outside of the custom draw function if the socket is connected. It seems is always expected behavior.